lib/pbench/test/functional/agent/cli/commands/triggers/test_register_tool_trigger.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+
+def test_pbench_register_tool_trigger_help():
+    command = ["pbench-register-tool-trigger", "--help"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"Usage: pbench-register-tool-trigger [OPTIONS]" in out
+    assert exitcode == 0
+
+
+def test_register_triggers(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+    trigger_default = pbench_run / "tools-v1-default" / "testhost.example.com"
+    trigger_default.mkdir(parents=True)
+    pidstat = trigger_default / "pidstat"
+    pidstat.touch()
+
+    # test-15
+    command = [
+        "pbench-register-tool-trigger",
+        "--group=default",
+        '--start-trigger="START DEFAULT"',
+        '--stop-trigger="STOP DEFAULT"',
+    ]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert exitcode == 0
+    trigger = pbench_run / "tools-v1-default" / "__trigger__"
+    assert trigger.exists()
+    assert b"tool trigger strings for start:" in out
```

Comment:
Can we read the contents of `__trigger__` and verify we see the expected values?